odk/odk.py

```diff
@@ -720,6 +720,7 @@ def runcmd(cmd):
     (out, err) = p.communicate()
     logging.info('OUT: {}')
```

Comment:
I think this should be `logging.info('OUT: {}'.format(out))` instead!

Reply:
Thanks, that looks more sensible now!